src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -793,34 +794,34 @@ export class AppComponent implements OnInit, AfterViewChecked {
     /**
      *
      */
-    keyEvent(e: KeyboardEvent): void {
-        const self = this;
-        let eString = this.shortcuts.buildEventString(e);
-        this.shortcuts.getEvent(eString).then((result) => {
+    async keyEvent(e: any): Promise<void> {
+        if (e.target.classList.contains('slick-cell')) {
+            $('#messages').css('user-select', 'none');
+            const self = this;
```

Comment:
If you're going to use self be consistent and use it everywhere - don't use this like you are below.